ProjectSidewalk/SidewalkWebpage
# On /labelmap/validate endpoint, delete label's validation if already exists for user

Resolves #3629

Before, the /labelmap/validate endpoint did not check if a user already had a validation for the given label. As a result, multiple validations could be created, leading to a variety of issues. This fix adds a check in the endpoint to see if a validation has already been made by the user. If yes, then it deletes the old validation and creates a new validation. If no, then it simply creates a new validation. This matches the behavior in `processValidationTaskSubmissions`, which did already have the check implemented correctly.

##### Video

https://github.com/user-attachments/assets/7ae4bf08-1d63-4552-992c-f5e957a8b65f

##### Testing instructions
1. Go to the gallery
2. Find a label and switch back and forth between agree and disagree. Previously, this would create multiple validations (see issue for more details).
3. Refresh and go to that same label. Instead of the previous multiple validations, you should see only one now.

##### Things to check before submitting the PR
